api.py

```diff
@@ -247,6 +247,8 @@ def post_message():
 
   db = get_db()
   message = dict(request.values)
+  if message['text'].strip() == "":
```

Comment:
Could you change `message['text'].strip()` to `request.values['text'].strip()` and put the changes on line 247, before the database connection is set up?

Reply:
Done.